samples/MvcAngular/AngularClient/app/app.js

```diff
@@ -0,0 +1,6 @@
+angular
+    .module('openiddemoclient',[
+        'ngRoute',
+        'ui.bootstrap',
+        'afOAuth2'
```

Comment:
It's this one:
https://github.com/JamesRandall/AngularJS-OAuth2

You can install it using `bower install angularjs-oauth2`